src/main/java/mat/server/util/MeasureTransferUtil.java

```diff
@@ -0,0 +1,41 @@
+package mat.server.util;
+
+import com.amazonaws.auth.InstanceProfileCredentialsProvider;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.PutObjectResult;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import mat.dto.MeasureTransferDTO;
+import org.apache.commons.lang3.StringUtils;
+
+public class MeasureTransferUtil {
+
+    public static AmazonS3 buildAwsS3Client() {
+        return AmazonS3ClientBuilder
+                .standard()
+                .withRegion(getSystemProperty(
+                        "MEASURE_TRANSFER_S3_AWS_REGION"))
+                .withCredentials(new InstanceProfileCredentialsProvider(true))
+                .build();
+    }
+
+    public static PutObjectResult uploadMeasureDataToS3Bucket(MeasureTransferDTO ob, String measureId)
```

Comment:
What's an `ob`?

Reply:
initially, this was a generic method to upload any object but later I changed it to measuretransfer. didn't change the variable though